scripts/benchmark/1.2.0/.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: cpp
```

Comment:
This can and should be `language: generic` since we handle setting up the C++ toolchain ourselves in the main `.travis.yml`